circleci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -uexo pipefail
+
+HOST_DMD_VER=2.068.2 # same as in dmd/src/posix.mak
+CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
+N=2
+
+case $CIRCLE_NODE_INDEX in
+    0) MODEL=64 ;;
+    1) MODEL=32 ;;
+esac
```

Comment:
Could you please enable two containers, s.t. 32-bit gets tested too?

Should be here https://circleci.com/gh/dlang/dmd/edit#parallel-builds